h/models/group.py

```diff
@@ -152,9 +152,6 @@ def groupid(self, value):
     )
     organization = sa.orm.relationship("Organization")
 
-    def __init__(self, **kwargs):
```

Comment:
I don't think this was doing anything but I might be wrong.

Reply:
Looks like nonsense to me!